### ISSUE TITLE
Tag Analytics for Artist

### ISSUE THREAD

**Issue by Einfach0lli** [BlenderKit/platform#11]
### Is your feature request related to a problem or weakness of addon? Please describe.

It would be nice to have deeper analytics for the assets we upload to Blenderkit. So we can write better descriptions or tags for the assets.

### Describe the solution you'd like

On the Website a third category in the middle of "my assets" and "wallet" an "analytics" category would be nice.
Where we can see which tags were the best for which model. Maybe a chart for every asset that shows how often it was downloaded the last 30 days or since upload.

**Comment by vilemduha** [comment]
@PetrDlouhy why me ;) does anybody have an idea this is feasible? Wouldn't it be better to move forward the analytics that have we been working on in the past?

**Comment by PetrDlouhy** [comment]
@vilemduha This needs to be specified, triaged, prioritized before we can start working on it. We can make some meeting about it.

There was some work done on the charts, but it would need much more work to finish it, and I am not sure if it would fit the needs of creators.
Also there are the marketplace statistics which might meet some of the criteria, but they keep breaking down and no one has the responsibility to fix them: https://www.blenderkit.com/asset-earnings-live/
